### PR TITLE
Update release notes in preparation for the 9.0.0 release.

### PR DESCRIPTION
**Pull request by Ouss4**

### CODE REVIEW

**Comment by patacongo** [comment]
There is also release informatino in Documentation/NuttX.html that needs to be updated correspondingly.  Also, all of the new boards and MCUs that are supported need to updated there.

**Comment by patacongo** [comment]
The feature list on the Confluence Wiki landing page should also be updated to match.

**Comment by hartmannathan** [comment]
I just noticed there is no mention of the stm32h747i-disco board, which is new in 9.0 (added just after the 8.2 release in November). I think it is still a work in progress: https://github.com/apache/incubator-nuttx/blob/master/boards/arm/stm32h7/stm32h747i-disco/README.txt.

**Comment by Ouss4** [comment]
> I just noticed there is no mention of the stm32h747i-disco board, which is new in 9.0 (added just after the 8.2 release in November). I think it is still a work in progress: https://github.com/apache/incubator-nuttx/blob/master/boards/arm/stm32h7/stm32h747i-disco/README.txt.

Thanks.  I added the board to the release notes and to the NuttX.html

**Comment by patacongo** [comment]
In NuttX.html, there are two tables, the first here: https://cwiki.apache.org/confluence/display/NUTTX/About#platforms and the second right after it.  I think those need to be updated too.

And the date.  It still says, "Last Updated: November 21, 2019" which was the date of the 8.2 release.

**Comment by adamfeuer** [comment]
@Ouss4 @patacongo @hartmannathan Do we need any other changes, or are the release notes good to go?

**Comment by hartmannathan** [comment]
> @Ouss4 @patacongo @hartmannathan Do we need any other changes, or are the release notes good to go?

LGTM.

**Comment by Ouss4** [comment]
@patacongo @adamfeuer @hartmannathan let's get it in please.

File: Documentation/NuttX.html [hunk 1]
```diff
@@ -1450,11 +1450,13 @@ <h2>Git Repository</h2>
...
 <p>
   In addition to the ever-changing GIT repository, there are frozen released versions of NuttX available.
-  The current release is NuttX 8.2.
-  NuttX 8.2 is the 133<sup>rd</sup> release of NuttX.
-  It was released on November 16, 2019, and is available for download from the
-  <a href="https://bitbucket.org/nuttx/nuttx/downloads/">Bitbucket.org</a> website.
-  Note that the release consists of two tarballs:  <code>nuttx-8.2.tar.gz</code> and <code>apps-8.2.tar.gz</code>.
+  The current release is NuttX 9.0.
+  NuttX 9.0 is the 134<sup>rd</sup> release of NuttX.
+  It was released on MMMM DD, YYYY, and is available for download from the
```
Comment (Ouss4) [inline Documentation/NuttX.html:1455]: When do you think it's a good time to change this?

Comment (adamfeuer) [reply Documentation/NuttX.html:1455]: @Ouss4 How far are we away from release? Do we have a target release date?

Comment (Ouss4) [reply Documentation/NuttX.html:1455]: We are aiming for the 30th of April.

Comment (adamfeuer) [reply Documentation/NuttX.html:1455]: @Ouss4 Can we merge now, then update the website with the HTML once we release? Or does the HTML get uploaded to the website automatically?

Comment (btashton) [reply Documentation/NuttX.html:1455]: The html gets inlined as part of a build step and then these files are uploaded to the wiki after the release.

The website is a sperate task where a page is generated from the release notes an a little metadata and then built by the CI and deployed

Comment (adamfeuer) [reply Documentation/NuttX.html:1455]: Thanks for the info @btashton. @Ouss4 Will you put in the 30 April 2020 date? Then we should be good to merge this PR, and can start working on the other things that are needed. If we need to adjust the date up or down later we can always create a one-line PR to fix the date.

Comment (Ouss4) [reply Documentation/NuttX.html]: Done.

Comment (adamfeuer) [reply Documentation/NuttX.html]: @Ouss4 Thanks!

@btashton @hartmannathan @patacongo Will one of you merge this when the checks finish ok?

**Pull request merged**